project/frontend/src/components/UserModal.js

```diff
@@ -0,0 +1,30 @@
+import './UserModal.css';
+
+import React from 'react';
+
+import Modal from '../components/Modal.js';
+
+function UserModal(props) {
+  return (
+    <Modal
+      open={props.isModalOpen}
+      onDismiss={props.toggleModal}
```

Comment:
nit: pull these variables out. So before return add `const {isModalOpen, toggleModal, user, signOut} = props` and then here and below you can just do `open={isModalOpen}` etc.